README.md

```diff
@@ -18,10 +18,11 @@ Cloud native serverless backend API for [UMCCR](https://umccr.org) [Data Portal
 aws sso login --profile=dev
 export AWS_PROFILE=dev
 make up
```

Comment:
Yap; `make up` will bring `mysql` container which takes awhile to be fully started. So that when `make start` to start Django development server, to make sure `mysql` container is ready for accepting connections.